src/storage/README.md

```diff
@@ -0,0 +1,86 @@
+# The Storage Layer
+
+The Storage layer is responsible of storing the data used and produced
+by the Consensus layer. Some of this data is "hot" and relates to the
+candidate chains and other data is "cold" and relates to the
+historical chain, following the principle described in [the `k`
+security parameter
+section](../consensus/chainsel.html#the-k-security-parameter).
```

Comment:
Is it correct to say data "produced" by the Consensus layer.  In a sense, this component receives data (in the form of blocks and transactions), and sometimes produces some other (blocks).

Reply:
Changed to "storing the data on behalf of the Consensus layer"